qemu/tests/hpt_miscellaneous.py

```diff
@@ -118,4 +118,11 @@ def run(test, params, env):
         session = vm.reboot(session)
         vm.verify_alive()
         verify_hpt(test, params, session, hpt_default)
-    vm.verify_kernel_crash()
+    if "migration" in params.get("sub_type"):
+        # do migration
+        mig_timeout = float(params.get("mig_timeout", "3600"))
```

Comment:
```suggestion
        mig_timeout = params.get_numeric("mig_timeout", "3600", float)
```

Reply:
@PaulYuuu , correct me if  I was wrong, mine is also acceptblein my opinions, do you mind tell me the reason ?

Reply:
Because the existing method can already handle it, no need to convert it yourself.